packages/get-platform/src/__tests__/getSSLVersion.test.ts

```diff
@@ -6,11 +6,26 @@ const describeIf = (condition: boolean) => (condition ? describe : describe.skip
 const ctx = jestContext.new().assemble()
 
 describeIf(process.platform === 'linux')('computeLibSSLSpecificPaths', () => {
-  // eslint-disable-next-line jest/no-identical-title
-  it('should not return an error', () => {
+  const OLD_ENV = process.env
+
+  afterAll(() => {
+    process.env = OLD_ENV // Restore old environment
+  })
```

Comment:
```suggestion
  const ORIGINAL_ENV = process.env

  beforeEach(() => {
    process.env = ORIGINAL_ENV // Restore old environment
  })
```

let's make sure each test starts with the fresh env and nothing can leak between them

Reply:
oh thanks, good catch :)